test/integration/cmds/merge/export.test.ts

```diff
@@ -39,7 +39,7 @@ describe('merge export command args validation', () => {
 
   it('should exit 1 when no source environment', done => {
     app()
-      .run('merge export --t target')
+      .run('merge export --se target')
```

Comment:
I think this should be `--te` (test title says "no source environment")

Reply:
👍🏻